logics/impls/flashloan_gateway.rs

```diff
@@ -20,7 +20,9 @@ pub const STORAGE_KEY: u32 = openbrush::storage_unique_key!(Data);
 #[derive(Debug)]
 #[openbrush::upgradeable_storage(STORAGE_KEY)]
 pub struct Data {
+    /// Flashloan Fee
```

Comment:
When I look at this value, the initial value is 9. What is the value? Is it a percentage?
It is better to explain not only the meaning of the value, but also what type of field the value is in.
```rust
// logics/impls/flashloan_gateway.rs
impl<T: Storage<Data>> Internal for T {
    default fn _initialize(&mut self, controller: AccountId) {
        self.data::<Data>().flashloan_premium_total = 9;
        self.data::<Data>().controller = controller;
    }
```